test/components/Payment_spec.js

```diff
@@ -15,9 +15,11 @@ const {
 const Payment = React.createFactory(PaymentComponent);
 
 describe('Payment', () => {
+  const closeOrder = () => {};
```

Comment:
for code dryness, we can create a setup function like the one in chef.js. I'm in the process of doing the same for orderdetails as i'm working through the set status back to open from ready to bill feature.